tests/integration_tests/test_chainermn.py

```diff
@@ -24,6 +28,7 @@
 
 
 STORAGE_MODES = ['new', 'common']
+PRUNER_TYPES = ['median', 'successive_halving']
```

Comment:
By directly passing pruners or their constructor calls, we could get rid of `_make_pruner` logic. For example:

```python
PRUNER_INIT_FUNC = [lambda: MedianPruner(), lambda: SuccessiveHalvingPruner()]
```

Reply:
It seems cool! I updated `test_chainermn.py` to reflect your idea (https://github.com/pfnet/optuna/pull/286/commits/661a7c2913565ed637a8ea8c6523299715950310).